src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -898,7 +898,7 @@ namespace OpenLoco
         }
         else
         {
-            if (!thoughtTypeHasFlags(thought.type, ThoughtTypeFlags::unk5 | ThoughtTypeFlags::unk6))
+            if (!thoughtTypeHasFlags(thought.type, ThoughtTypeFlags::unk4 | ThoughtTypeFlags::unk5))
```

Comment:
Given this seems to have been a mistake in the previous implementation, could this have led to a bug? For example, the wrong costs being applied? (Doesn't help that we don't exactly know what these flags represent, of course.)